dashboard/test/ui/features/star_labs/applab/libraries.feature

```diff
@@ -0,0 +1,73 @@
+# Maddie (6/12/2020) disabled in IE because "And I press keys" step does not work in IE.
+# https://github.com/code-dot-org/code-dot-org/pull/24646
+@no_ie
+@no_mobile
+Feature: Libraries
+
+  @as_student
+  Scenario: Publishing and unpublishing a library
+    Given I publish a basic library
+    Then I save the URL
+
+    # Check for library on /projects/libraries
+    Then I am on "http://studio.code.org/projects/libraries"
+    And I wait until element ".ui-test-library-table" is visible
+    And element ".ui-test-library-table td:contains('UntitledProject')" is visible
+
+    # Unpublish library
+    Then I navigate to the saved URL
+    And I wait for the page to fully load
+    Then I open the library publish dialog
+    And I click selector "#ui-test-unpublish-library" once I see it
+    And I wait until element "b:contains('Successfully unpublished your library')" is visible
+
+  Scenario: Adding and removing a library from a project
+    # Student1 publishes a library
+    Given I create a student named "Student1"
+    Given I publish a basic library
+    Then I save the channel id
+
+    # Student2 imports Student1's library
+    Given I create a student named "Student2"
+    And I start a new Applab project
```

Comment:
Would you be able to add a test for Gamelab too?

Reply:
i definitely can, but it looks like libraries aren't enabled in Gamelab, so i'm not sure how much value that would add. since you're OOO this week, i'm going to leave as-is for now and we can decide if we want to refactor this when you're back

Reply:
Oops. I think we forgot to enable libraries in Gamelab. Shouldn't block this PR, but probably should happen at some point.